seedfarmer/__main__.py

```diff
@@ -64,6 +64,13 @@ def version() -> None:
     Use only if bootstrapped with this qualifier""",
     required=False,
 )
+@click.option(
+    "--role-prefix",
+    default="/",
+    help="""An IAM path prefix to use with the seedfarmer roles.
+    Use only if bootstrapped with this path""",
+    required=False,
+)
```

Comment:
The changes for `apply`, `destroy`, `synth` seem undocumented?

Reply:
On it